frontend/javascripts/oxalis/view/action-bar/save_button.js

```diff
@@ -72,10 +72,10 @@ class SaveButton extends React.PureComponent<Props, State> {
         className={this.props.className}
       >
         {this.shouldShowProgress() ? (
-          <React.Fragment>
-            {Math.floor((progressInfo.processedActionCount / progressInfo.totalActionCount) * 100)}{" "}
+          <span style={{ marginLeft: 8 }}>
+            >{Math.floor((progressInfo.processedActionCount / progressInfo.totalActionCount) * 100)}{" "}
```

Comment:
Did you add the `>` deliberately?

Reply:
Oops, no, this wasn't deliberately. You are faster than the linter :P Thank you!